content/en/docs/Overview/_index.md

```diff
@@ -3,7 +3,7 @@ title: "Overview"
 linkTitle: "Overview"
 weight: 1
 description: >
-    This page introduces you to OLM and what you can achieve with it
+    What is OLM? why should I use it?
```

Comment:
```suggestion
    What is OLM? Why should I use it?
```

Reply:
I actually spent a while looking this up, either is valid.